bin/ch-run.c

```diff
@@ -1,5 +1,3 @@
-/* Copyright © Triad National Security, LLC, and others. */
-
```

Comment:
You don't have to remove these (they are still correct) -- but it may just be easier to do that and to omit any copyright statement from the code. Up to you. We use slightly different language: 

https://github.com/spack/spack/blob/develop/bin/spack#L4-L5